ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.less

```diff
@@ -76,13 +75,25 @@
     .is-split-button button,
     .is-button {
       height: @minimumButtonSize;
-      line-height: @minimumButtonSize;
       display: inline-block;
-      margin-left: @mediumSpacing;
+      text-align: middle;
```

Comment:
❓ Should this be center? I don't think middle is a valid value (but maybe browsers correct it for you)

Reply:
Nice catch. Removed because it wasn't having any effect